docs/releases/1.17-NOTES.md

```diff
@@ -79,6 +79,9 @@
 
 # Known Issues
 
+* Flannel users need to upgrade to Kubernetes versions 1.18.6, 1.17.9, 1.16.13 or higher to avoid significant performance degradation.
```

Comment:
```suggestion
* Flannel and Canal users need to upgrade to Kubernetes versions 1.18.6, 1.17.9, 1.16.13 or higher to avoid significant performance degradation.
```

Reply:
I was thinking to remove this for now. If we want this in just in master for a while, doesn't make sense to bundle it with release notes.